value_wrapper.go

```diff
@@ -318,6 +318,6 @@ func (valWarp ValueWrapper) String() string {
 		}
 		return fmt.Sprintf("[%s]", strings.Join(strs, ", "))
 	} else { // is empty
-		return "__EMPTY__"
+		return ""
```

Comment:
why change to empty string, It does not distinguish between a truly empty string and an empty value. How about to `<empty>` and null to `<null>`